src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -31,20 +31,18 @@ namespace Microsoft.PowerShell.Commands
 {
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
CmsMessage cmdlets thorough tests.Verify error when encrypting to wrong cert
```powershell
Expected 1, but got 0.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 211
211:         $errors.Count | Should -Be 1
```